travis.sh

```diff
@@ -282,8 +284,18 @@ function build_workspace() {
    # Console output fix for: "WARNING: Could not encode unicode characters"
    export PYTHONIOENCODING=UTF-8
 
-   # For a command that doesn’t produce output for more than 10 minutes, prefix it with travis_run_wait
-   travis_run_wait 60 --title "catkin build" catkin build --no-status --summarize
+   # Set compile flag to improve ccache hit rate
+   export CXXFLAGS="$CXXFLAGS -fdebug-prefix-map=$(pwd)=."
```

Comment:
I'm curious: How does this effect ccache? Isn't this a debugging option of gcc?

Reply:
That idea came from this: https://cppisland.wordpress.com/2018/12/30/ccache-quick-guide/

The basics is that with a Debug build there is a chance the hit rate is low because of paths in the debug symbols and this is one way to fix that.  This should have no affect on non-debug builds.

Reply:
Thanks for providing the link. However, their use case is different: They are building the same stuff in _different_ folders, while for us all files reside in the very same folder between build attempts. Hence, there shouldn't be a cache miss due to cwd.
Did you look into the log files generated with export `CCACHE_DEBUG=1` between to consecutive (local) builds?